README.md

```diff
@@ -1,74 +1,190 @@
+
+
+
+## 💻 Getting Started
+
 
-# Live Demo
-[Link to Live DEMO](https://calebchris000.github.io/Hello-World/)
 
-# Getting Started:
 To get a local copy up and running, follow these steps.
 
-# Prerequisites
+### Prerequisites
 
 In order to run this project you need:
-1. **Git**.
-2. **A text editor. (Visual Studio Code, Atom, Sublime Text.)**
-3. **NodeJS.**
 
 
-# Setup
-* Install git by running `sudo apt install git`
-* Install Visual Studio Code by running `sudo apt install visual-studio-code`
-* Install nodejs by running `sudo apt install npm`
-* Clone this repository to your desired folder: `git clone https://github.com/calebchris000/Hello-World.git`
+```sh
+ sudo apt install npm
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
 
-# Install
-Install this project with: linter
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
 
-Usage
-To run the project, execute the following command: `npm run index.html`
+### Install
 
+Install this project with: 
 
-# Run tests
+
+
+```sh
+  cd my-project
+  npm install
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+```sh
+  ssh server
+```
+
+
+### Run tests
 
 To run tests, run the following command:
-Deployment
+
+
+```sh
+  bin/npm test test/models/article_test.js
+```
+
+
+### Deployment
 
 You can deploy this project using:
 
 
-# Authors
+```sh
+npm deploy node/name-of-file
+```
+
+
+## 👥 Authors
+
+
+
+👤 **Caleb Nwaizu**
+
+- GitHub: [@githubhandle](https://https://github.com/calebchris000)
+- Twitter: [@twitterhandle](https://twitter.com/calebchris000)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/calebchris000)
+
+👤 **Nwaizu Joshua**
+
+- GitHub: [@githubhandle](https://github.com/josh2funny)
+- Twitter: [@twitterhandle](https://twitter.com/josh2funny)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/josh2funny)
+
+
+
+
+## 🔭 Future Features
+
+
+
+- [STYLING ] **[Add more styles to project]**
+- [EVENTS] **[Create a JS file that accepts input]**
+- [FUNCTIONS] **[Add a functionality that does things]**
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+
+
+
+## ⭐️ Show your support
+
+
+
+If you like this project, feel free to donate at our [PayPal link](paypal.com)
+
+
+
+## 🙏 Acknowledgments
+
+
+
+I would like to thank Microverse for helping me create this README file
+
+
+
+## ❓ FAQ
+
+
+- **[Would this project break my computer?]**
+
+ - [No. Why would it?]
 
-Caleb Chigozie Nwaizu.
+- **[Would this project be maintained in the future?]**
 
-# Future Features.
+ - [Yes it would]
 
-1. Implement styles on CSS
-2. Add elements to HTML
 
-Support our project by visiting microverse.org
 
-# Acknowledgements
-Nothing yet.
+## 📝 License
 
-# FAQ
-* Q. Would this project break my computer?
-   A. No. Why would it?
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Kindly make sure this link is not broken to make your README more professional :recycle: